book/03-hacking-atom/sections/06-iconography.asc

```diff
@@ -0,0 +1,27 @@
+=== Iconography
+
+Every so often you come to a point in time, where, to represent something, an icon is better off than words are.
+Since Atom is made by GitHub, their https://octicons.github.com/[Octicons] font is available for use.
+
+===== Usage
+
+Atom's usage of https://octicons.github.com/[Octicons] differs just a bit from the standard usage of them, that you'd find, say, in a web project.
```

Comment:
How about:

> Atom's usage of the Octicons differs just a bit from the [standard usage](https://octicons.github.com/usage/).

Since they're already linked above. Also mentioning "web project" feels not that necessary.